components/org.wso2.carbon.identity.sso.saml/src/main/java/org/wso2/carbon/identity/sso/saml/processors/SPInitLogoutRequestProcessor.java

```diff
@@ -128,6 +132,8 @@ public SAMLSSOReqValidationResponseDTO process(LogoutRequest logoutRequest, Stri
 
             // Validate signature of the logout request.
             if (logoutReqIssuer.isDoValidateSignatureInRequests()) {
+                //obtaining x509 Certificate
```

Comment:
use the correct comment format. Add space after //, start comment with capital and end with fullstop.

Reply:
Fixed: https://github.com/wso2-extensions/identity-inbound-auth-saml/pull/275